### PR TITLE
Link to Deep Learning Summer School Lecture

### PR DESCRIPTION
**Pull request by peterdsharpe**
Added link to Matthew's Deep Learning Summer School lecture, which I found very helpful.

### CODE REVIEW

**Comment by j-towns** [comment]
Nice, thanks! Can you change ‘Matthew’ to ‘Matt’ and then I’ll merge 😉.

**Comment by j-towns** [comment]
Also check out https://github.com/google/jax if you haven’t already!

**Comment by peterdsharpe** [comment]
Done!

Thanks for the heads up - jax looks awesome! What are the main differences between autograd and jax? Would one be better than the other for my application (gradient computations for engineering design optimization)?

**Comment by j-towns** [comment]
The main difference is that jax allows you to just-in-time (jit) compile your code, which can lead to speed ups and also allows for you to run code on GPU.

If you’re wondering which is best for your application you can easily try both Autograd and Jax, because their APIs are so similar. For differentiating numpy code you can just change between `import autograd.numpy as np` and `import jax.numpy as np` and similar for `grad`.

I recommend reading through the Jax readme for more detail on the other new features.

**Comment by peterdsharpe** [comment]
Awesome! Thanks so much for all your work on this project and for open-sourcing it - I really appreciate it!
